Makefile

```diff
@@ -104,6 +104,10 @@ build-koord-manager: ## Build koord-manager binary.
 build-koord-scheduler: ## Build koord-scheduler binary.
 	go build -o bin/koord-scheduler cmd/koord-scheduler/main.go
 
+.PHONY: build-runtime-manager
```

Comment:
can you add this step into the ci workflow?
https://github.com/koordinator-sh/koordinator/blob/ea8496c485835dac30f110c19c9270eff8c5c079/.github/workflows/ci.yaml#L90-L91

Reply:
done